### DISCUSSION TITLE
[Admob] Adaptive Banners

### DISCUSSION THREAD

**Discussion by JeffGuKang** [invertase/react-native-firebase#4276] [Ideas]
https://developers.google.com/admob/ios/banner/adaptive

They recommend to use `Adaptive Banners` instead of `Smart Banners`.

**Comment by dylancom** [comment]
Would like to see this integrated aswell! This is the new generation of mobile banners.

**Reply by mikehardy** [reply]
@dylancom we will happily accept and merge PRs - this not something that is a maintainer priority (and I personally believe AdMob should actually be peeled out of firebase since there are literally zero firebase-specific admob APIs) but if someone from the community contributes a clean PR, I'll shepherd it to release

**Reply by dylancom** [reply]
@mikehardy I understand. I just submitted a PR that adds adaptive banner support on iOS. Might need some help to get it also to work on Android.
